addons/clones/userscript.js

```diff
@@ -1,6 +1,7 @@
 export default async function ({ addon, console, msg }) {
   const vm = addon.tab.traps.vm;
 
+  let alsoShowOnProjectPage = addon.settings.get("projectpage");
```

Comment:
```suggestion
  let alsoShowOnProjectPage = addon.settings.get("projectpage");
```
"also" could probably be removed here too